components/scream/tests/preqx-nlev72/CMakeLists.txt

```diff
@@ -1,7 +1,4 @@
-set(SCREAM_TEST_SRCS
-  ${SCREAM_SRC_DIR}/share/util/scream_catch_main.cpp
-  scream_tests.cpp
-)
+INCLUDE (ScreamUtils)
```

Comment:
This test was supposed to be a system test, rather than a small unit test. For now it was just a place holder (not much happening) so it doesn't really matter, but in the future, this folder will not use CreateUnitTest.